state/accountsDB.go

```diff
@@ -1288,12 +1292,16 @@ func (adb *AccountsDB) snapshotUserAccountDataTrie(
 
 		stats.NewSnapshotStarted()
 
+		iteratorChannelsNew := &common.TrieIteratorChannels{
+			LeavesChan: nil,
```

Comment:
why nil?

Reply:
We do not need the leaves from dataTries on this channel in order to take a complete snapshot.

Reply:
re-checked, indeed we do not use it